ui-tests/src/test/java/org/openmrs/reference/page/AdministrationPage.java

```diff
@@ -16,7 +16,7 @@
 
     private static final String URL = "/admin/index.htm";
 
-    private static final By MANAGE_USERS = By.cssSelector("#legacyui-manageUsers a");
```

Comment:
Since this element have its id and we can as well use its ccselector, i think  we  can still maintain  **"#legacyui-manageUsers a** for code redability , since the full url  can easily change but id wont  change

Reply:
Thanks for the feedback !! Have wrapped the response in the commit with id `1107113`

Reply:
> i think we can still maintain **"#legacyui-manageUsers a** for code redability

This would be the best option among the alternatives. However, this element attribute value is only true when inspected via qa instance, and inspecting it via `demo.openmrs.org` & also on the` sdk local instance` the element property value is not true. In those two instances have mentioned the element `value property` is set to `#legacyui-mangeUsers a`. For code sustainability and quality assurance, I opted to set the property value to the suggested one since it hardly breaks with any server instance either



> full url can easily change

This ideally might not be true!!